python/lsst/drp/tasks/metadetection_shear.py

```diff
@@ -65,9 +65,14 @@ class MetadetectionShearConnections(PipelineTaskConnections, dimensions={"patch"
         dimensions={"patch", "band"},
     )
 
+    # TODO: make "name" configurable, as it will depend on
+    # the repo we are using
```

Comment:
These are already configurable from the pipeline yaml file which are specific to what repo we are running this against. `cal_ref_cat_2_2` is just the default value in the absence of a pipeline file.